<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>767</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="displayButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>QPushButton {
    background-color: beige;
    border-radius: 15px;
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #f0e68c;
}</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #D2B48C; /* Более коричневый оттенок бежевого */
    border: 2px solid #A0522D; /* Более контрастная окантовка */
    border-radius: 25px; /* Скругленные края */
    font-family: 'Arial'; /* Широкий шрифт */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный шрифт */
    padding: 10px 20px; /* Внутренние отступы */
    color: #8B4513; /* Более контрастный цвет шрифта */
}

QPushButton:hover {
    background-color: #C19A6B; /* Цвет при наведении */
}

QPushButton:pressed {
    background-color: #8B4513; /* Цвет при нажатии */
}</string>
       </property>
       <property name="text">
        <string>ТРАНСЛЯЦИЯ ВКЛ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cameraButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>QPushButton {
    background-color: beige;
    border-radius: 15px;
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #f0e68c;
}</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #D2B48C; /* Более коричневый оттенок бежевого */
    border: 2px solid #A0522D; /* Более контрастная окантовка */
    border-radius: 25px; /* Скругленные края */
    font-family: 'Arial'; /* Широкий шрифт */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный шрифт */
    padding: 10px 20px; /* Внутренние отступы */
    color: #8B4513; /* Более контрастный цвет шрифта */
}

QPushButton:hover {
    background-color: #C19A6B; /* Цвет при наведении */
}

QPushButton:pressed {
    background-color: #8B4513; /* Цвет при нажатии */
}</string>
       </property>
       <property name="text">
        <string>КАМЕРА ВКЛ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #D2B48C; /* Более коричневый оттенок бежевого */
    border: 2px solid #A0522D; /* Более контрастная окантовка */
    border-radius: 25px; /* Скругленные края */
    font-family: 'Arial'; /* Широкий шрифт */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный шрифт */
    padding: 10px 20px; /* Внутренние отступы */
    color: #8B4513; /* Более контрастный цвет шрифта */
}

QPushButton:hover {
    background-color: #C19A6B; /* Цвет при наведении */
}

QPushButton:pressed {
    background-color: #8B4513; /* Цвет при нажатии */
}</string>
       </property>
       <property name="text">
        <string>СПРАВКА О СОСТОЯНИИ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>189</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="disaplayLabel">
       <property name="text">
        <string>ТРАНСЛЯЦИЯ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="displayONButton">
       <property name="text">
        <string>ВКЛ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="displayOFFButton">
       <property name="text">
        <string>ВЫКЛ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cameraLabel">
       <property name="text">
        <string>КАМЕРА</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="cameraONButton">
       <property name="text">
        <string>ВКЛ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="cameraOFFButton">
       <property name="text">
        <string>ВЫКЛ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
